setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup, find_packages
+
+VERSION = 1
+
+setup(
```

Comment:
what do you suggest to put here? the exact same things that are listed in requirements.txt? this is an application, not a library, i only created setup.py because it was required by dh-virtualenv.